### ISSUE TITLE
APT whitelist request for sphinxsearch

### ISSUE THREAD

**Issue by md5** [travis-ci/travis-ci#3971]
We'd like to have this to take advantage of container-based builds and Bundler caching for https://github.com/thecartercenter/elmo

**Comment by BanzaiMan** [comment]
Sphinx is pre-installed in `/usr/local/sphinx-*/` for recent versions. Have you tried it?

**Comment by md5** [comment]
I should have tried it first. I didn't see it listed on this page, so I assumed it wasn't installed: http://docs.travis-ci.com/user/database-setup/

When I tried `apt-get install -y sphinxsearch` and saw that the package got installed, I took that as a confirmation that the Apt install should be whitelisted. I didn't think to look in `/usr/local/sphinx-*`.

Is the right Sphinx directory on the `PATH`, or would I need to add it?
